extensions/amp-analytics/0.1/vendors/webengage.js

```diff
@@ -17,9 +17,12 @@
 import {jsonLiteral} from '../../../../src/json';
 
 const WEBENGAGE_CONFIG = jsonLiteral({
+  'vars': {
+    'clientId': 'CLIENT_ID(we_luid)',
+  },
```

Comment:
Any reason for this variable? You can use ${clientId(wd_luid)} in your request without this, or you can use the macros version (CLIENT_ID(we_luid)) in the request as well.

Reply:
Make sense. I have made the changes at both the places. Could have caused some confusion.